.changeset/remove-deprecated-nunjucks-cli.md

```
@@ -0,0 +1,12 @@
+---
+"@asyncapi/generator": major
+---
+
+### Breaking Changes
+  * Nunjucks renderer removed — React is now the sole rendering engine. No need to specify render engine in config anymore
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)